clientv3/integration/kv_test.go

```diff
@@ -893,7 +893,7 @@ func TestKVLargeRequests(t *testing.T) {
 		expectError error
 	}{
 		{
-			maxRequestBytesServer:  1,
+			maxRequestBytesServer:  256,
```

Comment:
Why do we change this?

Reply:
Basically, if we keep the previous test data, the test cluster will fail to start. It is because the cluster will make several v3 raft requests like cluster version set during start. Old code will use v2 for starting server which will bypass the check. 
https://github.com/etcd-io/etcd/blob/1f8764be3b43448ccfd60706c42dab09b0bc6ed3/etcdserver/v3_server.go#L633-L635
Changing a larger `maxRequestBytesServer` will not affect the test behavior as long as `maxRequestBytesServer` is smaller then `valueSize`.